### PR TITLE
[IIachimov] Added IO Task

### PR DESCRIPTION
**Pull request by IIachimov**
Please review the task for I/O Streaming

### CODE REVIEW

File: iulianIachimov/src/main/java/iotask/utils/ReadFromDoc.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package iotask.utils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ReadFromDoc {
+  public static String[] readBufferedFile(File file) throws IOException {
```
Comment (solonarieugeniu) [inline iulianIachimov/src/main/java/iotask/utils/ReadFromDoc.java:8]: do you need an array of String? if not please change return to String, you could use StringWriter

Comment (IIachimov) [reply iulianIachimov/src/main/java/iotask/utils/ReadFromDoc.java:8]: I added method countWordInFileFromString(String word, Path path) directly in Count class

Comment (IIachimov) [reply iulianIachimov/src/main/java/iotask/utils/ReadFromDoc.java:8]: Sorry the method is countWordInFileFromString(String word, File file)

File: iulianIachimov/src/main/java/iotask/utils/Count.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    int result = 0;
+    try {
+      BufferedReader br = new BufferedReader(new FileReader(file));
+      String str;
+      while ((str = br.readLine()) != null) {
+        if (str.equalsIgnoreCase(word)) {
+          result++;
+        }
+      }
+      br.close();
```
Comment (asecu) [inline iulianIachimov/src/main/java/iotask/utils/Count.java]: can we put this in the finally block?

File: iulianIachimov/src/main/java/iotask/utils/WriteToDoc.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  public static void writeToFileNIO(Path path, String text) throws IOException {
+    try {
+      Files.write(path, text.getBytes(), StandardOpenOption.APPEND);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public static void writeFromMap(File file, String st) throws IOException {
+    FileWriter fw = new FileWriter(file);
```
Comment (asecu) [inline iulianIachimov/src/main/java/iotask/utils/WriteToDoc.java]: can we use try, catch?

File: iulianIachimov/src/main/java/iotask/utils/Count.java (hunk 2 above)
Comment (IIachimov) [reply iulianIachimov/src/main/java/iotask/utils/Count.java]: I added br.close to finally block

File: iulianIachimov/src/main/java/iotask/utils/WriteToDoc.java (hunk 3 above)
Comment (IIachimov) [reply iulianIachimov/src/main/java/iotask/utils/WriteToDoc.java]: I added try, catch, finally blocks

**Review by asecu — APPROVED** [review]

**Review by solonarieugeniu — APPROVED** [review]

**Pull request merged**